test/k8s/manifests/cnp-from-nodes.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "from-nodes"
```

Comment:
I would like to see `to-nodes` policy also tested similarly to this one.

Reply:
btw I wonder, whether I should drop this and extend cilium-cli connectivity tests (see https://cilium.slack.com/archives/C2B917YHE/p1700740103264699)

Reply:
I ended up removing this file in favor of cilium-cli tests.. please see https://github.com/cilium/cilium-cli/pull/2130